e2e-tests/tests/mtc_196_sets_test.go

```
@@ -0,0 +1,120 @@
+
+		runner.WorkDir = paths.TempDir
+
+		By("Run crane export/transform/apply pipeline")
+		log.Printf("Running crane pipeline for namespace %s\n", srcApp.Namespace)
+		Expect(RunCranePipelineWithChecks(runner, namespace, paths)).NotTo(HaveOccurred())
+		log.Printf("Crane pipeline completed for namespace %s\n", srcApp.Namespace)
+
+		By("Validate output directory contains DaemeonSet , ReplicaSet and StatefulSet resource manifest")
+		log.Println("Checking for DaemonSet resource manifest")
+		daemonset_pattern := filepath.Join(paths.OutputDir, "resources", namespace, "DaemonSet_*.yaml")
+		matches, err := filepath.Glob(daemonset_pattern)
```

golangci-lint (2.11.4)

[error] ineffectual assignment to err

(ineffassign)